scripts/run_regrid.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -x
+
+echo "Entered $0"
+MOM6REGRID=${MOM6REGRID:-$HOMEgfs}
+export EXEC_DIR=$MOM6REGRID/exec
+export USH_DIR=$MOM6REGRID/ush
+export COMOUT=$COMOUT
+export IDATE=$IDATE
+export ENSMEM=$ENSMEM
+export FHR=$fhr
+export DATA=$DATA
+export OCNFIXDIR=$OCNFIXDIR
+
+###### DO NOT MODIFY BELOW UNLESS YOU KNOW WHAT YOU ARE DOING #######
+#Need NCL module to be loaded:
+echo $NCARG_ROOT
+export NCL=$NCARG_ROOT/bin/ncl
+
+ls -alrt
+
+$NCL $USH_DIR/icepost.ncl > regrid_CICE.log 2>&1
```

Comment:
These scripts need to be added, the ice post.ncl and compost.ncl

Reply:
I have added icepost.ncl and ocnpost.ncl under the ush folder @ 382926d. Thanks!

Reply:
I updated the main PR comment to reflect the addition of the two ncl files.